testutil/testutil.go

```diff
@@ -49,32 +50,22 @@ func GetLocalHost() string {
 // host which might be drained e.g. in CI pipelines. This is useful to e.g.
 // create random passwords for tests where security is not a concern.
 func GetRandomString(chars int) string {
-	charset := []byte("abcdefghijklmnopqrstABCDEFGHIJKLMNOPQRSTUVWXYZ0123456789")
-
-	nchars := len(charset)
 	buffer := make([]byte, chars)
-	for i := range chars {
+	for i := range buffer {
 		//nolint:gosec // Using a weak random number generator on purpose to not drain entropy
-		buffer[i] = charset[rand.Intn(nchars)]
+		buffer[i] = charset[rand.Intn(len(charset))]
 	}
-
```

Comment:
What is the actual improvement here?

Reply:
Eliminated redundant variable - Removed the `nchars` variable by directly using len(charset) in the rand.Intn() call
Better variable naming - Using buffer in the range statement makes it explicit that we're iterating through the buffer indices
Minor performance improvement - Using a string constant instead of converting the string to a byte slice saves a memory allocation
The change from for i := range chars to for i := range buffer is not a bug fix but a readability improvement that makes the intent clearer.

Reply:
Thanks, the thing where I fully agree is the string vs. byte-slice part, the rest is subjective. ;-) Anyway, thanks for your work!